certified-connectors/Public360/apiDefinition.swagger.json

```diff
@@ -0,0 +1,377 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "Public 360",
+    "description": "\"Public 360 is a market leading ECM system in the Nordics. The Public 360 connector enables users to create flows that archive cases, documents and files into Public 360. The Public 360 connector will enable users to archive more and easier! That includes archiving social media posts, Outlook items or OneDrive documents.\"",

```

Comment:
The description should be formatted like this:

`"description": "Public 360 is a market leading ECM system in the Nordics. The Public 360 connector enables users to create flows that archive cases, documents and files into Public 360. The Public 360 connector will enable users to archive more and easier! That includes archiving social media posts, Outlook items or OneDrive documents.",`